elevate-user/constants/routes.js

```diff
@@ -22,6 +22,17 @@ module.exports = {
 				functionName: 'updateUser',
 			},
 		},
+		{
```

Comment:
where is reactivate OTP controller @sumanvpacewisdom

Reply:
i have included this